arbitrum-docs/devs-how-tos/bridge-tokens/how-to-bridge-tokens-custom-gateway.mdx

```diff
@@ -1,6 +1,5 @@
 ---
 title: 'How to bridge tokens via a custom gateway'
-sidebar_label: Custom gateway
```

Comment:
I've removed `sidebar_labels` from YAML frontmatter in many docs to make sidebar titling conventions easier to iterate on; we can work our way back to an autogenerated sidebar (& reintroduce this property in YAML frontmatter) as a future optimization.